frontend/app/src/next/hooks/use-crons.ts

```diff
@@ -102,7 +104,7 @@ export default function useCrons({
       return res.data;
     },
     onSuccess: () => {
-      listCronsQuery.refetch();
+      queryClient.invalidateQueries({ queryKey: ['cron:list'] });
```

Comment:
can we grab this query from `queries.cronList` or whatever it's called? maybe `queries.cronList.queryKey` or something?

Reply:
good idea, making a note to do this for all